manifests/rsyslog.pp

```diff
@@ -16,36 +16,28 @@
   String        $remote_syslog_server,
   Integer       $remote_syslog_server_port,
 ) {
-
-  $enabled = lookup("${module_name}::enable_session_log", Boolean)
-
-  if ($enabled) {
+  if ($profile_session_log::enable) {
     $ensure_parm = 'present'
 
-    $packages_defaults = {
-    }
-
     if ($forward_protocol == 'relp-tls') {
-      $required_pkgs = [ 'rsyslog-openssl' ]
+      $required_pkgs = ['rsyslog-openssl']
     } elsif ($forward_protocol == 'tcp-tls') {
-      $required_pkgs = [ 'rsyslog-gnutls' ]
+      $required_pkgs = ['rsyslog-gnutls']
     } else {
       fail("Invalid option given for forward_protocol: ${forward_protocol}")
     }
 
-    ensure_packages( $required_pkgs, $packages_defaults )
-
+    ensure_packages( $required_pkgs )
   } else {
     $ensure_parm = 'absent'
   }
 
-  file { '/etc/rsyslog.d/10-tlog-forward.conf':
+  file { '/etc/rsyslog.d/tlog-forward.conf':
     ensure  => $ensure_parm,
-    content => epp( "${module_name}/10-tlog-forward.conf.epp" ),
+    content => epp( "${module_name}/tlog-forward.conf.epp" ),
```

Comment:
Prefer pass parameters to epp() explicitly.

Reply:
Template updated to accept passed params.